### PR TITLE
BaSP-TG: Admin

### PR DESCRIPTION
**Pull request by jivescovo**
Maxi and I created all methods together during meet calls that have been logged in the minutas file.

### CODE REVIEW

**Comment by cami-diaz** [comment]
shouldn't the readme be updated?

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "node": "14.x"
   },
   "dependencies": {
-    "express": "^4.18.0",
+    "express": "^4.18.1",
```
Comment (cami-diaz) [inline package.json:28]: Is it necessary to change the version of express?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "backend-template",
-  "version": "1.0.0",
+  "name": "admin-features",
+  "version": "0.0.1",
```
Comment (cami-diaz) [inline package.json:3]: Is it necessary to change the version ?

File: src/resources/admins.js [hunk 3]
```diff
@@ -0,0 +1,161 @@
+const express = require('express');
+const fs = require('fs');
+const admins = require('../data/admins.json');
+
+const router = express.Router();
+
+// Add admin method
+router.post('/addAdmin', (req, res) => {
```
Comment (ignacioaristo) [inline src/resources/admins.js:8]: We could name it like `add` or `create`. We know that is an admin for the previous path.

File: src/resources/admins.js [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+        res.send('Admin deleted');
+      }
+    });
+  } else {
+    res.send('Non existent admin');
+  }
+});
+
+// Filter admins method
+router.get('/getListAdmin/:id/:name/:lastName/:email/:password', (req, res) => {
```
Comment (ignacioaristo) [inline src/resources/admins.js:61]: This get is weird, we should filter by 1 property in specific. For example with the `id` or `name`. Not all of them.

**Review by ignacioaristo — CHANGES_REQUESTED** [review]

**Review by ignacioaristo — DISMISSED** [review]

**Review by ignacioaristo — APPROVED** [review]
Good job

**Pull request merged**